frontend/projects/ui/src/app/app.component.ts

```diff
@@ -39,14 +39,6 @@ export class AppComponent implements OnDestroy {
   ) {}
 
   async ngOnInit() {
-    if (location.hostname !== 'localhost' && location.protocol === 'http:') {
```

Comment:
are we moving the redirect to the BE? or abandoning it alltogether?

Reply:
Abandoning. It's unnecessary and the code was problematic

Reply:
ok, could you relay why? just curious what changed @dr-bonez? for my own understanding

Reply:
1. It just didn't work in my testing
2. If it did work, the UX would be that after a period of time and with no indication why, the page would suddenly redirect/refresh
3. There was no catch block, so if the fetch failed, it would be an unhandled promise rejection.

We don't need to *force* people away from http. We just need to tell them why they should be using https

Reply:
Cool thanks for explaining. Those concerns could for sure be addressed in code, I definitely made a mistake not adding the catch block. In my testing, it only redirected once on initial load and worked great. Perhaps the lag over Tor would be abrupt. I think we had wanted to do this for a few reasons, rather than making the user manually change, but understood its not ideal to force the behavior.